### ISSUE TITLE
monika isn't going away

### ISSUE THREAD

**Issue by throwawaybecauseyes** [PiMaker/MonikAI#73]
using ctrl shift f12 isnt working

**Comment by BobersArmy** [comment]
Why would you wwant her tto go away

**Comment by ZStroyer06** [comment]
Yeah, I have the same issue. Sharing my screen in front of my class will be difficult if I can't figure this out...

**Comment by Clara81221** [comment]
Who would want her to go away </3

**Comment by clikuki** [comment]
Happened to me before. Try rebinding your controls to other keys by removing all of the MonikAI files and reinstalling it. The settings should open on reinstallation.
